content/master/concepts/environment-configs.md

```diff
@@ -371,7 +308,94 @@ The environments selected by
 {{<hover label="maxMatch" line="18">}}matchLabels{{</hover>}} are then merged
 into any other environments listed in the 
 {{<hover label="maxMatch" line="7">}}environmentConfigs{{</hover>}}.
+
 <!--
 TODO: Add Policies
-TODO: Add webhook validations
 -->
+
+
+## Patching with EnvironmentConfigs
+
+When Crossplane creates a composite resource, Crossplane creates a unique copy  
+of the EnvironmentConfig for the composite resource. 
```

Comment:
```suggestion
When Crossplane reconciles a composite resource, Crossplane merges all the specified EnvironmentConfigs into an in-memory environment.
```

Reply:
I know "reconciles" is the technically correct work but I don't like it, it's too much about the guts of k8s/crossplane.

Would it accurate to say "creates or updates" a composite resource?

Reply:
Yeah, could be fine